layouts/partials/analytics.html

```diff
@@ -19,11 +19,11 @@
 
         <!-- GoatCounter -->
         {{ with .goatCounter }}
-            <script
-            data-goatcounter="https://{{ .code }}.goatcounter.com/count"
-            async
-            src="//gc.zgo.at/count.js"
-            ></script>
+            {{ if .selfHosted }}
+                <script data-goatcounter="https://{{ .url }}/count" async src="https://{{ .url }}/count.js"></script>
+            {{ else }}
+                <script data-goatcounter="https://{{ .code }}.goatcounter.com/count" async src="//gc.zgo.at/count.js"></script>
+            {{ end }}
```

Comment:
This will break all existing users site. Please, use this:

```html
{{ with .goatCounter }}
     {{ $server := (printf "https://%s.goatcounter.com" .code) }}
     {{ if .selfHosted }}
         {{ $server = .url}}
     {{ end }}
     <script data-goatcounter="https://{{ $server }}/count" async src="//gc.zgo.at/count.js"></script>
{{ end }}
```